src/Stratis.Bitcoin/P2P/ISelfEndpointTracker.cs

```diff
@@ -7,6 +11,15 @@ namespace Stratis.Bitcoin.P2P
     /// </summary>
     public interface ISelfEndpointTracker
     {
+        /// <summary>Update external IP address and peer score of the node.</summary>
+        /// <paramref name="ipEndPoint">The endpoint to add.</paramref>
+        /// <paramref name="suppliedEndPointIsFinal">Whether the ipEndPoint supplied should be marked final on the endpoint tracker.</paramref>
+        /// <paramref name="ipEndPointPeerScore">Peer score of the ipEndPoint supplied. Default value of 1.</paramref>
```

Comment:
misunderstanding here

the parameters should be done with `<param>` as you see elsewhere, but if inside of that description you refer to DIFFERENT parameter, you use `<paramref>` so for example here:

`<param name="ipEndPointPeerScore">Peer score of the <paramref name="ipEndPoint"/> supplied. Default value of 1.</paramref>`